cli.js

```diff
@@ -130,6 +131,7 @@ var api = new Api({
 	serial: cli.flags.serial,
 	require: arrify(cli.flags.require),
 	cacheEnabled: cli.flags.cache !== false,
+	disablePowerAssert: cli.flags.powerAssert === false,
```

Comment:
Yeah, the CLI part should be kept as-is. `--no` is the best way to handle negative flags.